.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
```

Comment:
is the plan that in the future there will no longer be an additional manual step to publish to npm? that's nice!

note that sometimes we need to release an npm version without tagging it as the latest. here's how we currently do that with semaphore: https://www.notion.so/workos/a52c9c5a71ce48578e7e3e26fc82b386?pvs=4#76ad07d3c81b4e2aa8946eeb430524bd

we'll need a replacement (can be hacky)

Reply:
Good to know, I'll see if there's a less manual way of handling this or otherwise try to support the same flow.